partition/roles/systemd-networkd/tasks/main.yaml

```diff
@@ -29,7 +38,7 @@
   when: item.name != 'lo'
   notify:
   - restart systemd-networkd
-  - restart systemd-udev-trigger
+  - rename interfaces
```

Comment:
This handler does not exist?

Reply:
There's two handlers that listen for that event:

https://github.com/metal-stack/metal-roles/blob/11aeaf21135c0f0e5b3788f87721074012090b4f/partition/roles/systemd-networkd/handlers/main.yaml#L32
https://github.com/metal-stack/metal-roles/blob/11aeaf21135c0f0e5b3788f87721074012090b4f/partition/roles/systemd-networkd/handlers/main.yaml#L38

The point for this is:
The renaming of interfaces does not take effect while they're still up; so the following deployment steps that depend on the interface names would fail. 
With this procedure the interfaces are taken offline and the renaming gets triggered.

Reply:
Thanks for explaining. I was not aware of the `listen` keyword.